services/uploads/serverless.yml

```diff
@@ -364,6 +382,94 @@ resources:
                 - !Sub ${QAUploadsBucket.Arn}/*
               Sid: DenyUnencryptedConnections
 
+    ClamAVSecurityGroup:
+      Type: AWS::EC2::SecurityGroup
+      Properties:
+        GroupDescription: Security group for ClamAV daemon
+        VpcId: ${self:custom.vpcId}
+        SecurityGroupIngress:
+          - IpProtocol: tcp
+            FromPort: 3310
+            ToPort: 3310
+            SourceSecurityGroupId: ${self:custom.sgId}
+
+    ClamAVInstanceProfile:
+      Type: AWS::IAM::InstanceProfile
+      Properties:
+        Path: '/delegatedadmin/developer/'
+        Roles:
+          - !Ref ClamAVInstanceRole
+
+    ClamAVInstanceRole:
+      Type: AWS::IAM::Role
+      Properties:
+        Path: '/delegatedadmin/developer/'
+        PermissionsBoundary: !Sub 'arn:aws:iam::${AWS::AccountId}:policy/cms-cloud-admin/ct-ado-poweruser-permissions-boundary-policy'
+        RoleName: !Sub 'clamavdVm-${sls:stage}-ServiceRole'
+        AssumeRolePolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+            - Effect: Allow
+              Principal:
+                Service: ec2.amazonaws.com
+              Action: sts:AssumeRole
+        Policies:
+          - PolicyName: ClamAVInstancePolicy
+            PolicyDocument:
+              Version: '2012-10-17'
+              Statement:
+                - Effect: Allow
+                  Action:
+                    - logs:CreateLogGroup
+                    - logs:CreateLogStream
+                    - logs:PutLogEvents
+                  Resource: '*'
+
+    ClamAVInstance:
+      Type: AWS::EC2::Instance
+      Condition: IsDevValProd
+      Properties:
+        InstanceType: t3.medium
+        ImageId: ami-0c7217cdde317cfec # Ubuntu 22.04 LTS
+        IamInstanceProfile: !Ref ClamAVInstanceProfile
+        KeyName: mojo # for testing
+        NetworkInterfaces:
+          - AssociatePublicIpAddress: true
+            DeviceIndex: '0'
+            GroupSet:
+              - !Ref ClamAVSecurityGroup
+              - ${self:custom.sgId}
+            SubnetId: !Sub ${self:custom.publicSubnetA}
+        Tags:
+          - Key: Name
+            Value: clamavd-${sls:stage}
+          - Key: mcr-vmuse
+            Value: clamavd
+        UserData:
+          Fn::Base64: !Sub |
+            #!/bin/bash
+            apt-get update
+            apt-get install -y clamav clamav-daemon
+
+            echo '${self:custom.authorizedKeys}' > /home/ubuntu/.ssh/authorized_keys
+            chown ubuntu:ubuntu /home/ubuntu/.ssh/authorized_keys
+            chmod 600 /home/ubuntu/.ssh/authorized_keys
+
+            # Install clamav 1.0.4
+            wget https://www.clamav.net/downloads/production/clamav-1.0.4.linux.x86_64.deb
```

Comment:
is there a way to just always grab `latest` here? Or will we need to keep updating this on the reg?

Reply:
Not really, they don't host an apt server for us to just grab `latest` from, we have to pull the package manually and install it like this if we want a recent version. For whatever reason the main apt repo has an older version of ClamAV and I thought it best to use the current LTS.

We'll definitely want to make sure we keep up with patches over time.